NuRadioReco/detector/response.py

```diff
@@ -418,13 +497,14 @@ def _calculate_time_delay(self):
         mask = np.all([195 * units.MHz < freqs, freqs < 250 * units.MHz], axis=0)[:-1]
         time_delay1 = np.mean(time_delay[mask])
 
-        # fit the unwrapped phase with a linear function
-        popt = np.polyfit(freqs, np.unwrap(phase), 1)
-        time_delay2 = -popt[0] / (2 * np.pi)
+        # This alternative calculation is only meaningful if group delay is ~ constant over the whole frequency range (which is the case for most cables)
```

Comment:
So you decided to throw it out? I don't follow the logic. It is the case, so we don't calculate it to check for deviations?

Reply:
This alternative calculation (which is only well motivated for cable) was only used as a cross-check and caused a lot of warnings. Furthermore, the calculation itself is not used for physics results but only for consistency checks.

Reply:
Ok. I just wanted to understand why someone once thought that this was useful and now we simply remove it.

Reply:
The one was me :D